example/matrix/CMakeLists.txt

```diff
@@ -10,11 +10,13 @@ project(Matrix)
 
 find_package(YARP REQUIRED)
 
+find_package(GSL REQUIRED)
```

Comment:
FindGSL.cmake was added in CMake 3.2 , I propose to increase the cmake minimum required version to 3.5 as we will soon do this for all YARP.